docs/getting-started-guides/rkt/notes.md

```diff
@@ -0,0 +1,132 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Notes on Different UX with rkt container runtime
+
+### Doesn't support ENTRYPOINT + CMD feature
+
+To run a Docker image, rkt will transfer it into App Container Image (ACI) format first.
+However, during the transformation, the `ENTRYPOINT` and `CMD` are concatentated to construct ACI's `Exec` field.
+This means after the transformation, we are not able to replace only `ENTRYPOINT` or `CMD` without touching the other part.
+So for now, users should specify the **full executable path + arguments** in either the container spec's `Command` or `Args`.
```

Comment:
Specifying full path + args in `Args` seems a big strange to me. Shouldn't we tell the users to set both `Command` and `Args`?